plexapi/base.py

```diff
@@ -283,6 +283,8 @@ def fetchItems(self, ekey, cls=None, container_start=None, container_size=None,
 
             results.extend(subresults)
 
+            last_container_size = container_size
```

Comment:
Isn't it okay to just move `container_start += container_size` up here before the `maxresults` logic below?

Reply:
Yep @JonnyWong16 that was my first thought too. Then I thought, what if later calculations based on container_size needed to be considered, but I'm probably over-worrying.

Reply:
I think just keep it simple and move it above.

Reply:
Agreed, changes committed. New test passes, please excuse poor internet connection and consequently slow responses.